src/server/chunk_server/chunk_server_impl.cc

```diff
@@ -125,5 +132,59 @@ ChunkServerImpl::GetChunkServerProtocolClient(
   }
 }
 
+bool ChunkServerImpl::ReportToMaster() {
+  // Prepare the report chunk server request
+  ReportChunkServerRequest request;
+
+  // Add the chunk server information to the request
+  auto chunk_server = request.mutable_chunk_server();
+  auto chunk_server_location = chunk_server->mutable_location();
+
+  chunk_server_location->set_server_hostname(
+      config_manager_->GetServerHostname(chunk_server_name_));
+  chunk_server_location->set_server_port(
+      config_manager_->GetServerPort(chunk_server_name_));
+
+  // TODO(bmokutub): Use std::filesystem to get the available disk space.
+  // Setting to 20GB for now, fix this.
+  chunk_server->set_available_disk_mb(/*available_disk_mb=*/20 * 1024);
+
+  // We also need to tell the master all the chunks we have, if any.
+  auto all_chunks_metadata =
+      FileChunkManager::GetInstance()->GetAllFileChunkMetadata();
+
+  LOG(INFO) << "Found " + std::to_string(all_chunks_metadata.size()) +
```

Comment:
nit. you can directly use `LOG(INFO) << "Found" << all_chunk_metdata.size() << ...`